cpp/src/components/connectivity.cu

```diff
@@ -70,26 +70,22 @@ std::enable_if_t<std::is_signed<IndexT>::value>
   gdf_column* labels = table->get_column(0);
   gdf_column* verts = table->get_column(1);
 
-  CUGRAPH_EXPECTS(graph != nullptr, "Invalid API parameter");
-    
-  CUGRAPH_EXPECTS(graph->adjList != nullptr, "Invalid API parameter");
-    
-  CUGRAPH_EXPECTS(row_offsets_(graph) != nullptr, "Invalid API parameter");
+  CUGRAPH_EXPECTS(graph != nullptr, "Invalid API parameter: Graph is NULL");
+  CUGRAPH_EXPECTS(graph->adjList != nullptr, "Invalid API parameter: Graph is empty");
 
-  CUGRAPH_EXPECTS(col_indices_(graph) != nullptr, "Invalid API parameter");
+  CUGRAPH_EXPECTS(row_offsets_(graph) != nullptr, "Invalid API parameter: Graph is empty");
+  CUGRAPH_EXPECTS(col_indices_(graph) != nullptr, "Invalid API parameter: Graph is empty");
   
   CUGRAPH_EXPECTS(labels->data != nullptr, "Invalid API parameter");
-
   CUGRAPH_EXPECTS(verts->data != nullptr, "Invalid API parameter");
   
   auto type_id = graph->adjList->offsets->dtype;
-  CUGRAPH_EXPECTS( type_id == GDF_INT32 || type_id == GDF_INT64, "Unsupported data type");
-  
-  CUGRAPH_EXPECTS( type_id == graph->adjList->indices->dtype, "Unsupported data type");
+  CUGRAPH_EXPECTS( type_id == GDF_INT32 || type_id == GDF_INT64, "Unsupported data type: Graph must be int32 or int64");
+  CUGRAPH_EXPECTS( type_id == graph->adjList->indices->dtype, "Unsupported data type: Graph must be int32 or int64");
   
   //TODO: relax this requirement:
   //
-  CUGRAPH_EXPECTS( type_id == labels->dtype, "Unsupported data type");
+  CUGRAPH_EXPECTS( type_id == labels->dtype, "Unsupported data type: labels->dtype must be int32 or int64");
```

Comment:
This error message is misleading, if type_id is Int32 and labels->dtype is Int64, this assertion will fail even if data types are Int32 or Int64.